targets/targets.json

```diff
@@ -6563,6 +6563,27 @@
             "CONFIG_GPIO_AS_PINRESET"
         ]
 },
+    "S1SBP6A": {
+        "inherits": ["Target"],
+        "core": "Cortex-M4F",
+        "supported_toolchains": ["GCC_ARM", "ARMC6"],
```

Comment:
can these be on new lines same as `device_has` ?

Also line 6571

Reply:
Thank you. I'll change it.